recipes/dlib/win_png.patch

```diff
@@ -0,0 +1,51 @@
+--- dlib/CMakeLists.txt
++++ dlib/CMakeLists.txt
+@@ -303,48 +303,9 @@
+          # installed on this system.  
+          if (PNG_FOUND)
+             set(CMAKE_REQUIRED_LIBRARIES ${PNG_LIBRARY})
+-            CHECK_FUNCTION_EXISTS(png_create_read_struct LIBPNG_IS_GOOD)
```

Comment:
This was failing and I have no idea why - so just disabling it made it link properly.